tests/command_line/test_stills_process.py

```diff
@@ -132,7 +135,23 @@ def test_sacla_h5(dials_data, tmp_path, use_mpi, in_memory=False):
     assert geometry_path.is_file()
 
     # Write the .phil configuration to a file
-    tmp_path.joinpath("process_sacla.phil").write_text(sacla_phil % geometry_path)
+    phil_path = tmp_path / "process_sacla.phil"
+    with open(phil_path, "w") as f:
+        f.write(sacla_phil % geometry_path)
+
+        if known_orientations:
+            known_orientations_path = os.path.join(
+                sacla_path, "SACLA-MPCCD-run266702-0-subset-known_orientations.expt"
+            )
+            # assert os.path.isfile(known_orientations_path)
```

Comment:
Need to uncomment this and be sure it passes, meaning dials.data  should include `SACLA-MPCCD-run266702-0-subset-known_orientations.expt`